### PR TITLE
Update sdk to 3.36

### PR DESCRIPTION
**Pull request by corecontingency**
See pull request on io.github.Hexchat [here](https://github.com/flathub/io.github.Hexchat/pull/8)

### CODE REVIEW

File: io.github.Hexchat.Plugin.Perl.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
     "prepend-path": "/app/extensions/Perl/bin"
   },
   "modules": [
+    "shared-modules/dbus-glib/dbus-glib-0.110.json",
+    {
+      "name": "dbus-glib"
```
Comment (TingPing) [inline io.github.Hexchat.Plugin.Perl.json]: This empty block can be removed.

Comment (TingPing) [inline io.github.Hexchat.Plugin.Perl.json]: This empty block can be removed also.

File: io.github.Hexchat.Plugin.Perl.json [hunk 2]
```diff
@@ -12,6 +12,10 @@
     "prepend-path": "/app/extensions/Perl/bin"
   },
   "modules": [
+    "shared-modules/dbus-glib/dbus-glib-0.110.json",
```
Comment (TingPing) [inline io.github.Hexchat.Plugin.Perl.json]: Actually I think we can probably get rid of this entirely, just build hexchat with `-Dwith-dbus=false`

File: io.github.Hexchat.Plugin.Perl.json [hunk 3]
```diff
@@ -52,6 +52,7 @@
         "--buildtype=release",
         "-Dwith-lua=false",
         "-Dwith-gtk=false"
```
Comment (TingPing) [inline io.github.Hexchat.Plugin.Perl.json]:
```suggestion
        "-Dwith-gtk=false",
```

File: .gitmodules [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[submodule "shared-modules"]
```
Comment (TingPing) [inline .gitmodules]: You can now remove this. (assuming it builds)

Comment (corecontingency) [reply .gitmodules]: Haha, will I ever not forget the comma?

**Comment by TingPing** [comment]
Thanks!

**Pull request merged**